packages/components/carousel/src/use-carousel.ts

```diff
@@ -79,6 +84,11 @@ export const useCarousel = (
     handleIndicatorHover(index)
   }, THROTTLE_TIME)
 
+  const isTwoLengthShow = (index: number) => {
```

Comment:
This function only applies in indicators, maybe better to rename it to make it more explicit.